screenpy/actions/tryto.py

```diff
@@ -0,0 +1,79 @@
+"""
+try/accept logic using screenplay pattern
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from screenpy.pacing import the_narrator
+from screenpy.speech_tools import get_additive_description
+
+if TYPE_CHECKING:
+    from screenpy import Actor
+    from screenpy.protocols import Performable
+
+
+class TryTo:
+    """
+    TryTo perform followup action if the first one fails.
+
+    Examples::
+
+        the_actor.will(TryTo(DoAction()).otherwise(DoDifferentAction())
+
+        the_actor.will(
+            TryTo(CheckIfOnDomain(URL())).otherwise(Open.their_browser_on(URL())
+        )
+    """
+
+    Second: tuple[Performable, ...]
```

Comment:
I don't think the `First` and `Second` properties shoudl be capitalized, it feels weird. Classes are capitalized, properties shouldn't be unless they're also classes? Since this is a list of Performables, maybe `first_act` and `second_act`?

Reply:
Good call.